.travis.yml

```diff
@@ -4,8 +4,19 @@ scala:
   - 2.11.8
   - 2.12.1
 
+cache:
+  directories:
+    - $HOME/.ivy2/cache
+    - $HOME/.sbt/boot
```

Comment:
I/someone should update http://www.scala-sbt.org/0.13/docs/Travis-CI-with-sbt.html#%28Experimental%29+Reusing+Ivy+cache

Also could you include the `before_cache` part too please, for correctness:

```yml
before_cache:
  - find $HOME/.ivy2/cache -name "ivydata-*.properties" -print -delete
  - find $HOME/.sbt        -name "*.lock"               -print -delete
```